content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -47,6 +50,16 @@ service:
       exporters: [logging]
 ```
 
+To directly emit traces to a backend without utilizing a collector, 
+specify the endpoint field in the Kubernetes TracingConfiguration with the desired trace backend address. 
```

Comment:
Readers might think TracingConfiguration is an API. Try “tracing configuration file”?

Reply:
Use “tracing configuration file”. Much less likely to confuse.

(BTW, our configuration files don't hold _resources_ in the Kubernetes API sense, because you don't use HTTP to interact with them)

Reply:
I have updated per comments, thanks for your input ~

Reply:
@sftim Tim, I have updated per comments, thanks!
Could you please help review again and approve if ok?